docs/feature-gates.md

```diff
@@ -323,3 +324,12 @@ Refer to this [document](service-loadbalancer.md) for more information.
 #### Requirements for this Feature
 
 This feature is currently only supported for Nodes running Linux.
+
+### TrafficControl
+
+`TrafficControl` enables a CRD API for Antrea that manages and manipulates the
+transmission of Pod traffic. It allows users to mirror or redirect specific
+traffic originating from specific Pods or destined to specific Pods to a local
+network device or a remote destination via various tunnels. It provides full
```

Comment:
How about changing "provides" to "enables", or "enables a monitoring solution to get full visibility..."